src/terminal/parser/stateMachine.cpp

```diff
@@ -163,7 +187,7 @@ static constexpr bool _isCsiDelimiter(const wchar_t wch) noexcept
 // - True if it is. False if it isn't.
 static constexpr bool _isCsiParamValue(const wchar_t wch) noexcept

```

Comment:
I plan to replace `isCsiParamValue` with `isNumericParamValue` (same for delimiter) in next PR, if it's OK with you guys.

I've always prefer more generic namings.

Reply:
Yeah I was thinking replacing them all and then I realize it may not be appropriate. But I already replaced some of them. So it is somewhere in the middle and it is for sure strange. I'll replace them all later.